ahk/script.py

```diff
@@ -14,10 +14,12 @@
 import subprocess
```

Comment:
It doesn't look like there's any substantive changes in this file related to the change described in the PR?

If that's the case, please omit formatting-only changes to files that are otherwise unedited.

Thanks!

Reply:
I format this file with black and args is same as you tell me before

Reply:
Thanks for formatting the file with Black, I appreciate that. 

But other than formatting changes, are there any functional changes in _this_ file?

If there's not any functional changes, let's leave it out to keep the file's history clean, even if it needs to be formatted :)

The purpose being that if someone looks at a file's history and see it was changed in connection with this PR, it could be confusing to try to figure out why `script.py` changed to support tooltip/traytip.

Reply:
But  I commit these changes with the name "👨‍💻 Code improvement" not "Tooltip..." or something like this.